recipes/newick/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "newick" %}
```

Comment:
should we maybe also call this package python-newick?

Reply:
OH! it is already there and it is called `python-newick`. thank you! Will clos ethe PR